modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -4,36 +4,52 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation, jre
 
-Recommended minimum hardware is:
+If you plan to install Mule and run it on premises, review these minimum hardware and software requirements before you install.
 
-* 2GHz CPU or 1 virtual CPU in virtualized environments
-* 1GB of RAM
-* 4GB of storage
+== Minimum Hardware Requirements
+
+* 2GHz CPU or 1 virtual CPU in virtualized environments.
+* 1GB of RAM.
+* 4GB of storage.
 
 Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Required Software
+
+Verify that you use a supported version of Java before you install Mule.
+
+[%header,cols="20a,75a"]
+|===
+|Software |Version
+|JDK | JDK 1.8.0 (1.8.0_202 recommended), IBM JDK 1.8, OpenJDK 8 (latest update recommended)
+|===
+
+== Supported Software
+
+[NOTE]
+Mule runs on any operating system that runs a supported Java Runtime Environments. Mule runtime is tested against the following software and operating system resulting in a valid and correct functionality.
+Other software and operating system might work with Mule but have not been tested. It is recommended to run Mule on software and operating system versions that have been validated.
```

Comment:
"It is recommended to run Mule on software and operating system versions that have been validated." -> "MuleSoft recommends that you run Mule on tested software and operating system versions." 

The suggested change above avoids passive voice. It uses "tested" instead of "validated," since the former is more commonly used in this context.

Reply:
Updated.